mbt_gym/gym/backtesting.py

```diff
@@ -0,0 +1,53 @@
+import gym
+import numpy as np
+import pandas as pd
+from mbt_gym.gym.TradingEnvironment import (
+    TradingEnvironment, 
+    ASSET_PRICE_INDEX, 
+    INVENTORY_INDEX, 
+    CASH_INDEX
+)
+from mbt_gym.agents.Agent import Agent
+from mbt_gym.gym.helpers.generate_trajectory import generate_trajectory
+
+
+def get_sharpe_ratio(env: gym.Env, agent: Agent, risk_free_rate: float = 0.099):
+    """
+    The Annualized Sharpe Ratio is calculated as: 
+                Sharpe_Ratio = sqrt(num_steps)*(Returns  - Risk Free Rate)/(Std of Return)
+    It measures the reward in relation to risk. 
+    """
+    assert env.num_trajectories == 1, "Backtesting is applied on a single trajectory"
+    obs, _, _ = generate_trajectory(env, agent)
+    portfolio_values = (obs[:, CASH_INDEX, :] + obs[:, INVENTORY_INDEX, :] * obs[:, ASSET_PRICE_INDEX, :]).squeeze()
+    return_pcts = np.diff(portfolio_values, 1)/portfolio_values[1:]
```

Comment:
What happens if `portfolio_values[1:]` has values very close to zero or even zero? Maybe you could raise a Warning? Similarly, there should probably be a warming if `return_pcts.mean()` takes a negative value as the Sharpe behaves weirdly then.